wd_helper.au3

```diff
@@ -1018,7 +1024,7 @@ EndFunc   ;==>_WD_ElementOptionSelect
 ; Link ..........:
 ; Example .......: No
 ; ===============================================================================================================================
-Func _WD_ElementSelectAction($sSession, $sSelectElement, $sCommand)
+Func _WD_ElementSelectAction($sSession, $sSelectElement, $sCommand, $vParameters = Null)
```

Comment:
A few questions about `$vParameters = Null` --

- Why a variant? Is there a plan to use this for other purposes?
- Did you consider using `$vParameters = Default` instead? This would be more inline with the rest of the functions.

Reply:
We can change to `$aParameters` .
About `Default` yes it can be changed.

Reply:
Take a look on: [$vParameters > $aParameters](https://github.com/Danp2/au3WebDriver/pull/212/commits/c87649a2235d4ba6f74cc916e07ae1807a1cb9b9)

Reply:
This doesn't even compile. Please validate your code before submitting for review. 😉

Reply:
Fixed: [$UBOUND_COLUMS typo](https://github.com/Danp2/au3WebDriver/pull/212/commits/5a31fd0b1f176562200dfc6162f04868c0f7e015)